airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -152,16 +152,16 @@ def conn_extras(self):
             extras = {}
         return extras
 
-    def _get_field(self, field_name):
+    def _get_field(self, field_name, default=None):
         if field_name.startswith('extra_'):
             raise ValueError(
                 f"Got prefixed name {field_name}; please remove the 'extra__kubernetes__' prefix "
                 f"when using this method."
             )
         if field_name in self.conn_extras:
-            return self.conn_extras[field_name] or None
+            return self.conn_extras[field_name] or default
```

Comment:
I know this is existing code, but using truthy here is unfortunate. A problem for another day though.

Reply:
i think truthy is good here right cus you don't want to return empty string right?